apps/etterlatte-saksbehandling-ui/client/src/App.tsx

```diff
@@ -19,13 +20,15 @@ function App() {
         <div className="app">
           <BrowserRouter basename="/">
             <HeaderWrapper />
-            <Routes>
-              <Route path="/" element={<Oppgavebenken />} />
-              <Route path="/oppgavebenken" element={<Oppgavebenken />} />
-              <Route path="/person/:fnr" element={<Person />} />
-              <Route path="/behandling/:behandlingId/*" element={<Behandling />} />
-              <Route path="*" element={<Navigate to="/" replace />} />
-            </Routes>
+            <ErrorBoundary>
```

Comment:
Noen spesiell grunn til at du flytter denne?

Reply:
Har bare cherry-picka inn en commit fra en annen pr, men ja, den er flyttet for å få med øverste linje